Formula/shtools.rb

```diff
@@ -0,0 +1,44 @@
+class Shtools < Formula
+  desc "Spherical Harmonic Tools"
+  homepage "https://shtools.github.io/SHTOOLS/"
+  url "https://github.com/SHTOOLS/SHTOOLS/archive/v4.7.tar.gz"
+  sha256 "2d6f426dbdeb8e3a16b1cd5ea5dd5db228bdb76f438abebcccd15e43096e8d37"
+  license "BSD-3-Clause"
+  head "https://github.com/SHTOOLS/homebrew-shtools.git"
+
+  depends_on "fftw"
+  depends_on "gcc"
+
+  def install
+    ENV.deparallelize
```

Comment:
Why can't it build in parallel?

Reply:
We are working on that. There is a complicated race situation occuring where it breaks 1/number-of-processors times. The easiest was to turn this off for now. It turns out that a new PR came in yesterday that might fix this, but I need to test it. This would likely be removed in the next release.